static/scss/_breakpoint-slicer.sass

```diff
@@ -0,0 +1,8 @@
+// Default values
+@import breakpoint-slicer/variables
+
+// Helper functions
+@import breakpoint-slicer/helper-functions
+
+// The magic!
+@import breakpoint-slicer/mixins
```

Comment:
This seem to be an external dependency - to download instead of commiting a copy of the files in git? https://github.com/lolmaus/breakpoint-slicer